travis: add page (#4744)
tldr-pages/tldr: pages/common/travis.md

# travis

> > Command line client to interface with a Travis CI service.

Reviewer: Perhaps this will read better?
```suggestion
> Command line client to interface with Travis CI.
```

> More information: <https://github.com/travis-ci/travis.rb>.

> - Output the client version:

Reviewer 2: ```suggestion
- Display the client version:
```

`travis version`

> - Authenticate against the API and store the token:

Reviewer 2: ```suggestion
- Authenticate the CLI client against the server, using an authentication token:
```

`travis login`

> - List repositories the user has certain permissions on:

Reviewer: Perhaps this will read better?
```suggestion
- List repositories the user has permissions on:
```

`travis repos`

> - Encrypt values for the .travis.yml:

Reviewer: Perhaps this will read better?
```suggestion
- Encrypt values in `.travis.yml`:
```

Also, what's the token value for?

Author: I have committed your suggestion. The token value is the data to be encrypted. An example is: `travis encrypt SOMEVAR="secretvalue"` This is to store encrypted environment variables in Travis.

`travis encrypt {{token}}`

> - Generate a .travis.yml and enable the project:

Reviewer: Perhaps this will read better?
```suggestion
- Generate a `.travis.yml` file and enable the project:
```

Also, what precisely is meant by "enabling the project"? Enable it to do what?

Author: See https://github.com/travis-ci/travis.rb#enable. Enabling a project activates it on Travis CI.

Reviewer: Ah I see. Sounds it has technical meaning then! The above suggestion should be fine then.
